projects/plugins/jetpack/_inc/client/traffic/style.scss

```diff
@@ -200,3 +200,13 @@
 		margin-top: 0.75rem;
 	}
 }
+
+.jp-stats-odyssey-badge {
+	color: $white;
+	background-color: $green-primary;
```

Comment:
<img width="1065" alt="image" src="https://user-images.githubusercontent.com/4044428/216165088-5165013b-b72e-4201-8f2b-da5f28687d7a.png">

Overall, styling looks good! However, we probably want to stick to the design specifications and use `$blue-medium` (#3582c4) as the background color.

cc @jeffgolenski

Reply:
Sure thing. I'll make this change today and we can update the colour again later if needed.